src/backend/distributed/master/master_node_protocol.c

```diff
@@ -69,10 +69,9 @@ int ShardReplicationFactor = 1; /* desired replication factor for shards */
 int ShardMaxSize = 1048576;     /* maximum size in KB one shard can grow to */
 int ShardPlacementPolicy = SHARD_PLACEMENT_ROUND_ROBIN;
 
-
+static List * GetTableReplicaIdentityCommand(Oid relationId);
```

Comment:
@hanefi realized that we're not setting the replica identities after shard moves. So, say a shard have a replica identity full, moving the shard leads to losing that information. 

That might lead to interesting bugs if a shard without a primary key is moved twice. In that case, during the second move, logical replication rejects INSERT/UPDATE/DELETEs, which is annoying.  

@hanefi is going to open an issue, I'm just wondering did we see a specific reason not to do it or just forgot. @furkansahin any memories?